CaseStudyProductListing/Dashboard/Model/Articles.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  Articles.swift
+//  CaseStudyProductListing
+//
+//  Created by Atul Vishnoi on 14/07/23.
+//
+
+import Foundation
+
+struct ArticlesResponse: Codable {
+    let status: String
```

Comment:
Should we not declare these Properties as optional?

Reply:
yup agree